### PR TITLE
Updated TextVectorization

### PR DESCRIPTION
**Pull request by chunduriv**
Changed import statement 
from tensorflow.keras.layers.experimental.preprocessing import TextVectorization
to
from tensorflow.keras.layers import TextVectorization

Removed below note section, since experimental.preprocessing.TextVectorization moved under tensorflow.keras.layers
{
      "cell_type": "markdown",
      "metadata": {
        "id": "HSDO9JZYrK-O"
      },
      "source": [
        "Note: The Preprocessing APIs used in this section are experimental in TensorFlow 2.3 and subject to change."
      ]
    },

and Changed from
preprocessing.TextVectorization
to
TextVectorization

### CODE REVIEW

**Review by MarkDaoust — APPROVED** [review]

File: site/en/tutorials/load_data/text.ipynb [hunk 1]
```diff
@@ -352,22 +352,13 @@
...
-      ]
-    },
     {
       "cell_type": "markdown",
       "metadata": {
         "id": "N6fRti45Rlj8"
       },
       "source": [
-        "Next, you will standardize, tokenize, and vectorize the data using the `preprocessing.TextVectorization` layer.\n",
+        "Next, you will standardize, tokenize, and vectorize the data using the `layers.TextVectorization` layer.\n",
```
Comment (8bitmp3) [inline site/en/tutorials/load_data/text.ipynb:361]: Small suggestion:

```suggestion
        "Next, you will standardize, tokenize, and vectorize the data using the `tf.keras.layers.TextVectorization` layer.\n",
```

This way, the API docs should be auto-linked when the notebook is viewed on tensorflow.org.

LMKWYT

Comment (MarkDaoust) [reply site/en/tutorials/load_data/text.ipynb:361]: The rule for the auto-links is "at least one `.` in the quoted path". The link will work either way.

Comment (8bitmp3) [reply site/en/tutorials/load_data/text.ipynb:361]: OK!

Comment (chunduriv) [reply site/en/tutorials/load_data/text.ipynb:361]: @MarkDaoust , @8bitmp3  - Should i need to do any thing from my end for the recommend suggestion above? Please let me know. Thanks!

**Review by MarkDaoust — APPROVED** [review]

**Comment by 8bitmp3** [comment]
The notebook didn't pass the style guide tests (first person -> second person), so I introduced the fixes. Thanks for the PR!

**Pull request merged**